tests/integration/test_auth.py

```diff
@@ -38,9 +97,27 @@ def test_user_login(client):
     response = client.json.post(url, json.dumps(credentials))
     assert response.status_code == 200
     expected_keys = [
-        'id', 'email', 'first_name', 'last_name', 'auth_token'
+        'id', 'first_name', 'last_name', 'email', 'gender', 'tshirt_size', 'ticket_id', 'phone_number',
+        'is_core_organizer', 'is_volunteer', 'date_joined', 'is_active', 'is_staff', 'is_superuser', 'auth_token'
+    ]
+    assert set(expected_keys).issubset(response.data.keys())
+
+
+def test_user_login_with_incorrect_credentials(client):
+    url = reverse('auth-login')
+    u = f.create_user(email='test@example.com', password='test')
+    credentials = {
+        'email': u.email,
+        'password': 'wrong_password'
+    }
+    response = client.json.post(url, json.dumps(credentials))
+    assert response.status_code == 400
+    expected_keys = [
+        'errors', 'error_type'
     ]
     assert set(expected_keys).issubset(response.data.keys())
+    assert response.data['error_type'] == 'WrongArguments'
```

Comment:
Check the name of the field for which error is appearing here.

Reply:
In this case API is not sending the name of the field for which the error is appearing. As it was default, so I didn't alter it.

Reply:
Sounds good 👍